scripts/samples/add-file-to-collection.sh

```diff
@@ -15,9 +35,7 @@ echo "$fname"
 EST_HOST="https://shuttle-4.estuary.tech"
 #EST_HOST="https://shuttle-5.estuary.tech"
 
-COLLECTION=""
-
+COLLECTION="185d7ffc-85d8-4b73-86bb-1ca6419bc10d"
```

Comment:
while this collection always be available? why not also allow passing in a collection uuid?